src/app/profile/spaces/space.service.spec.ts

```diff
@@ -0,0 +1,89 @@
+import { cloneDeep } from 'lodash';
```

Comment:
We are following the same style pattern that the Angular team recommends. Please list the @ angular imports first, then a space followed by the external imports, and another space and internal imports.